saleor/plugins/webhook/plugin.py

```diff
@@ -2104,8 +2124,12 @@ def shipping_zone_metadata_updated(
             WebhookEventAsyncType.SHIPPING_ZONE_METADATA_UPDATED, shipping_zone
         )
 
-    def _trigger_staff_event(self, event_type, staff_user):
-        if webhooks := get_webhooks_for_event(event_type):
+    def _trigger_staff_event(self, event_type, staff_user, webhooks=None):
+        if (
```

Comment:
In many places, you are making the same `if` condition. 
It is split on multiple lines so it is not super easy to read.
What about a helper method or function, that will make return a correct webhook list?
```python
    def _get_webhooks_for_event(self, event_type, webhooks):
        if webhooks is not None:
            return webhooks
        return get_webhooks_for_event(event_type)
    def _trigger_attribute_value_event(self, event_type, attribute_value):
        if webhooks := self._get_webhooks_for_event(event_type, webhooks=webhooks):
```